<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftwareKeyboard</class>
 <widget class="QWidget" name="SoftwareKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1127</width>
     <height>291</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_1">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string notr="true">ESC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string notr="true">1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string notr="true">2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string notr="true">3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string notr="true">4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string notr="true">5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string notr="true">6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string notr="true">7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string notr="true">8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string notr="true">9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string notr="true">-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string notr="true">^</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string notr="true">\</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string notr="true">BS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string notr="true">Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_17">
       <property name="text">
        <string notr="true">Q</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_18">
       <property name="text">
        <string notr="true">W</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_19">
       <property name="text">
        <string notr="true">E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_20">
       <property name="text">
        <string notr="true">R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_21">
       <property name="text">
        <string notr="true">T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_22">
       <property name="text">
        <string notr="true">Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_23">
       <property name="text">
        <string notr="true">U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_24">
       <property name="text">
        <string notr="true">I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_25">
       <property name="text">
        <string notr="true">O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_26">
       <property name="text">
        <string notr="true">P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_27">
       <property name="text">
        <string notr="true">@</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_28">
       <property name="text">
        <string notr="true">[</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_29">
       <property name="text">
        <string notr="true">RET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_30">
       <property name="text">
        <string notr="true">Ctrl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_31">
       <property name="text">
        <string notr="true">A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_32">
       <property name="text">
        <string notr="true">S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_33">
       <property name="text">
        <string notr="true">D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_34">
       <property name="text">
        <string notr="true">F</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_35">
       <property name="text">
        <string notr="true">G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_36">
       <property name="text">
        <string notr="true">H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_37">
       <property name="text">
        <string notr="true">J</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_38">
       <property name="text">
        <string notr="true">K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_39">
       <property name="text">
        <string notr="true">L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_40">
       <property name="text">
        <string notr="true">;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_41">
       <property name="text">
        <string notr="true">:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_42">
       <property name="text">
        <string notr="true">]</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_43">
       <property name="text">
        <string notr="true">Shift</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_44">
       <property name="text">
        <string notr="true">Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_45">
       <property name="text">
        <string notr="true">X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_46">
       <property name="text">
        <string notr="true">C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_47">
       <property name="text">
        <string notr="true">V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_48">
       <property name="text">
        <string notr="true">B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_49">
       <property name="text">
        <string notr="true">N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_50">
       <property name="text">
        <string notr="true">M</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_51">
       <property name="text">
        <string notr="true">,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_52">
       <property name="text">
        <string notr="true">.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_53">
       <property name="text">
        <string notr="true">/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_54">
       <property name="text">
        <string notr="true">\</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_55">
       <property name="text">
        <string notr="true">↑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_56">
       <property name="text">
        <string notr="true">Shift</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_57">
       <property name="text">
        <string notr="true">Fn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_58">
       <property name="text">
        <string notr="true">Alt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_59">
       <property name="text">
        <string notr="true">Windows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_60">
       <property name="text">
        <string notr="true">JP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_61">
       <property name="text">
        <string notr="true">Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_62">
       <property name="text">
        <string notr="true">Ctrl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_63">
       <property name="text">
        <string notr="true">Alt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_64">
       <property name="text">
        <string notr="true">Fn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_65">
       <property name="text">
        <string notr="true">←</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_66">
       <property name="text">
        <string notr="true">↓</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_67">
       <property name="text">
        <string notr="true">→</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
